pkg/GAPJulia/JuliaInterface/julia/gap_to_julia.jl

```diff
@@ -9,11 +9,23 @@ The parameter `recursion_dict` is meant to preseve egality
 of converted objects and should never be given by the user.
 """
 
+## Default for conversion:
+## Do not convert Julia types, as long as type matches
+## If you have a GAP list of Julia Tuples, this function
+## enables conversion of that list to a list of type
+## Array{Tuple,1}.
```

Comment:
How about this instead:
```suggestion
## As an example why this is useful, suppose you have a GAP list x (i.e., an
## object of type MPtr) containing a bunch of Julia tuples. Then this method
## enables conversion of that list to a Julia array of type Array{Tuple,1},
## like this:
##    gap_to_julia(Array{Tuple{Int64},1},xx)
## This works because first the gap_to_julia method with signature
## (::Type{Array{T,1}}, :: MPtr) is invoked, with T = Tuple{Int64}; this then
## invokes gap_to_julia recursively with signature (::Tuple{Int64},::Any),
## which ends up selecting the method below.
```

Reply:
I added that suggestion